pywebpush/__init__.py

```diff
@@ -143,10 +143,14 @@ def encode(self, data):
             'body': encrypted,
         })
 
-    def send(self, data, headers={}, ttl=0):
+    def send(self, data, gcm_key=None, reg_id=None, headers={}, ttl=0):
```

Comment:
nit: move new args to the end of the parameter list, so that unlabeled parameters aren't broken by accident.